src/css/textarea.less

```diff
@@ -10,12 +10,17 @@
 
   .mq-textarea *, .mq-selectable {
     .user-select(text);
-
     position: absolute; // the only way to hide the textarea *and* the
     clip: rect(1em 1em 1em 1em); // blinking insertion point in IE
-
     resize: none; // hotfix: https://code.google.com/p/chromium/issues/detail?id=355199#c1
+  }
+
+  //remove system cursor on iPad
+  .mq-textarea textarea:focus {
+    text-indent: -9999em;
+  }
 
+  .mq-selectable {
     width: 1px; // don't "stick out" invisibly from a math field,
     height: 1px; // can affect ancestor's .scroll{Width,Height}
```

Comment:
I haven't been able to reproduce that. I guess you'd need a mathquill field
that's smaller than 13px by 13px, which at least for Desmos hasn't arisen.
Setting to width or height to anything under (I think) 13px triggered this
chrome display bug for me.





On Wednesday, July 8, 2015, Jason Merrill <notifications@github.com> wrote:

> In src/css/textarea.less
> <https://github.com/mathquill/mathquill/pull/422#discussion_r34185276>:
>
> >
> > +  .mq-selectable {
> >      width: 1px; // don't "stick out" invisibly from a math field,
> >      height: 1px; // can affect ancestor's .scroll{Width,Height}
>
> According to these comments, setting width and height was fixing real
> bugs. Did we verify that those are no longer necessary for some reason?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mathquill/mathquill/pull/422/files#r34185276>.
>